src/js/page_sections/Menu/Submenu.js

```diff
@@ -1,16 +1,23 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
 
 import SubmenuItem from 'js/page_sections/Menu/SubmenuItem';
 import ExternalLink from 'js/modules/ExternalLink';
 import I18nNavLink from 'js/modules/I18nNavLink';
 import ArrowRightSVG from 'js/svg/ArrowRight';
 
-const Submenu = ({id, openSection, theme, handleMenuToggle}) => (
-  <ul className={`coa-Submenu
-      ${ id > 4 ? `coa-Submenu--align-right` : '' }
-      ${ openSection === id ? 'coa-Submenu--open' : '' }
-    `}
+const getSubmenuClassnames = (intl, id, openSection) => {
+  const base = 'coa-Submenu';
+  const shouldAlignRight = (intl.locale === 'ar' && id < 5) ||  (intl.locale !== 'ar' && id > 4);
```

Comment:
This magic number took me a sec to figure out. Maybe a const could help clear that up (though it's similar to the code that was there, so 👍 for maintaining existing style 😎 )

Reply:
I second the constant part of ^